AdminWebsite/AdminWebsite/Mappers/AddNotificationRequestMapper.cs

```diff
@@ -316,21 +317,26 @@ public static class AddNotificationRequestMapper
             {
                 notificationType = NotificationType.StaffMemberDemoOrTest;
                 parameters.Add(NotifyParams.StaffMember, $"{participant.FirstName} {participant.LastName}");
-            }
+            }            
             else if (participant.UserRoleName.Contains(RoleNames.Judge, StringComparison.InvariantCultureIgnoreCase))
             {
-                if (hearing.IsJudgeEmailEJud())
-                {
-                    notificationType = NotificationType.EJudJudgeDemoOrTest;
-                }
-                else
-                {
-                    notificationType = NotificationType.JudgeDemoOrTest;
-                    parameters.Add(NotifyParams.CourtroomAccountUserName, participant.Username);
-                }
+               judgeEmail = hearing.GetJudgeEmail();
+               if (string.IsNullOrEmpty(judgeEmail))
+               {
+                   return null;
+               }
+                    if (hearing.IsJudgeEmailEJud())
+                    {
+                        notificationType = NotificationType.EJudJudgeDemoOrTest;
+                    }
```

Comment:
Should this come before the null check? As judgeEmail will always be null for eJud. Or does the GetJudgeEmail() method handle getting it from the judge participant in that case